Парсер + EDA

Batmad01 opened the pull request:
Добавление Парсера Хабра Хабра

Batmad01 committed:
Create dataset.md

Данные, полученные с помощью парсинга Хабра

Batmad01 committed:
Update dataset.md

Batmad01 committed:
Парсер статей Хабра Хабра

Парсер в двух версиях - тетрадка .ipynb и питоновский скрипт .py

Chewye commented:
1. Присутствуют артефакты # In[29]:, вероятно, из ноутбука. Их нужно убрать.
Было бы полезно использовать линтер, так как беглым взглядом заметны несоответствия стандарту PEP 8.
2. Следует добавить комментарии к функциям и аннотацию типов.
3. Если мы планируем переиспользовать функции и методы (а, вероятно, так и есть), их нельзя запускать без конструкции if __name__ == '__main__'. Лучше вынести запуск в отдельный файл.
4. Рассмотри возможность оформления кода в виде класса, а также добавь (измени) функции (методы) для парсинга статьи по ссылке (инференс?) и для парсинга новых статей (обновление датасета).

Batmad01 committed:
Парсер_v2

- Добавлено описание всех функций и аннотация типов
- С помощью линтера скрипт приведен к стандарту PEP 8
- Убраны артефакты ноутбука из скрипта
- Исходный скрипт разделен на скрипт со всеми функциями и скрипт запуска парсера

Batmad01 committed:
Починка кодировки

Batmad01 committed:
EDA

Batmad01 committed:
Delete EDA.ipynb

Batmad01 committed:
Delete README.md

Batmad01 committed:
Delete checkpoints.md

Batmad01 committed:
Rename dataset.md to habr_data_urls.md
